pages/components/inputs/input.css

```diff
@@ -0,0 +1,29 @@
+.input {
+  position: relative;
+  margin: 1em;
+}
+
+.input > input {
+  border-radius: 0.3em;
+  border-style: solid;
+  border-width: 2px;
+  width: 15em;
+  height: 2em;
+}
+
+.input > div {
+  position: absolute;
+  top: -0.5em;
+  left: 1.5em;
+  background-color: #fff;
+}
+
+.success {
+  border-color: palegreen;
+}
+.warn {
+  border-color: rgb(255, 150, 14);
+}
+.error {
+  border-color: red;
+}
```

Comment:
you can create variables for these colors

Reply:
ok